avocado/core/messages.py

```diff
@@ -101,7 +101,8 @@ def handle(self, message, task, job):
                     'base_path': base_path,
                     'task_path': task_path,
                     'time_start': message['time'],
-                    'name': task.identifier}
+                    'name': task.identifier,
+                    'task_category': task.category}
```

Comment:
This commit could be part of the previous, IMO.

Reply:
Is it a strict requirement? For me, splitting them makes more sense, but I can merge if that is a problem.

Reply:
No, just a nitpicking. :)